lib/Models/Catalog/CatalogItems/OpenDataSoftCatalogItem.ts

```diff
@@ -631,8 +631,8 @@ export default class OpenDataSoftCatalogItem
           ([field, value]) => {
             // geoPoint2dFieldName will return a JSON object - spilt lat/lon columns
             if (field === this.geoPoint2dFieldName && isJsonObject(value)) {
-              cols.lat[index] = `${value.lat}` ?? "";
-              cols.lon[index] = `${value.lon}` ?? "";
+              cols.lat[index] = `${value.lat}`;
```

Comment:
this changes the behaviour so nullish lat will become "undefined" which we might not want in the table?

To preserve it becoming an empty string can do:
```suggestion
             cols.lat[index] = `${value.lat ?? ""}`;
```

Reply:
hmm.. afaik understand this preserves the current behaviour.
i.e if `lat` is undefined then, both before and after, the resulting value will be `"undefined"`.

I didn't want to change any behaviour in this PR so I went with the trivial fix.

Reply:
whoops my bad you're right